cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -643,6 +643,57 @@ <h2>Fees</h2>
             {% endfor %}
         {% endif %}
     </dl>
+    <h2>Rewards and perks</h2>
+    <dl class="m-list m-list__card-details">
+        <dt>Rewards</dt>
+        <dd>
+            {% if card.rewards %}
+            <ul>
+                {% for reward in card.rewards %}
+                    <li>
+                        {{ 'Cash back' if reward == 'Cashback rewards' }}
+                        {{ 'Travel-related rewards' if reward == 'Travel-related rewards' }}
+                        {{ card.other_rewards if reward == 'Other rewards' }}
```

Comment:
Do we want to make a similar change on the list view? That could also wait until that design is further along.

Reply:
I'm thinking we'll see how people react to the details page in usability testing, then we can carry over any changes that tested well to other pages.